<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolunteerDlg</class>
 <widget class="QDialog" name="VolunteerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>234</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="first_name_lbl">
       <property name="text">
        <string>First Name:</string>
       </property>
       <property name="buddy">
        <cstring>first_name</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="first_name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="last_name_lbl">
       <property name="text">
        <string>Last Name:</string>
       </property>
       <property name="buddy">
        <cstring>last_name</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="last_name"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="phone_lbl">
       <property name="text">
        <string>Phone Number(s):</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="email_lbl">
       <property name="text">
        <string>Email Address:</string>
       </property>
       <property name="buddy">
        <cstring>email</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="email"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="phone_home">
       <property name="placeholderText">
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="phone_mobile">
       <property name="placeholderText">
        <string>Mobile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>first_name</tabstop>
  <tabstop>last_name</tabstop>
  <tabstop>phone_home</tabstop>
  <tabstop>phone_mobile</tabstop>
  <tabstop>email</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VolunteerDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VolunteerDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
